pygmt/src/grdtrack.py

```diff
@@ -37,10 +37,10 @@ def grdtrack(points, grid, newcolname=None, outfile=None, **kwargs):
     r"""
     Sample grids at specified (x,y) locations.
 
-    Reads one or more grid files and a table (from file or an array input; but
+    Reads a grid file and a table (from file or an array input; but
```

Comment:
> As grid= can't be repeated in the same function call and it doesn't accept a list of grids, this pull request changes the documentation to state that the grdtrack accepts a single grid.

Technically someone smart enough could figure out that doing `grid="grid1.nc -Ggrid2.nc -Ggrid3.nc"` would work, but it's more of a hacky solution :upside_down_face: